coffee-module/coffee-module-configdoc/src/main/java/hu/icellmobilsoft/coffee/module/configdoc/ConfigDoc.java

```diff
@@ -65,4 +65,20 @@
      */
     String since() default "";
 
+    /**
+     * (Optional) {@code true} if the field is a startup parameter. Default {@code false}
```

Comment:
ezzel a  `{@code true}` résszel mit szeretttél volna elérni?

Reply:
Az `exclude` paraméternél láttam így használva és az alapján használtam itt is, reprezentálja, hogy mit jelent a true eset amiért használjuk és az alapértelmezett a false a végén. Nem így kell?

Reply:
oksa, így értem köszi, habár én mondatba fogalmaznám hogy ez az érték true ha... de így is okés